draft-specifications/CapTP Specification.md

```diff
@@ -770,143 +789,136 @@ result.
 
 ## [`desc:handoff-give`](#desc-handoff-give)
 
-You should have an overview of handoffs before reading this section, please read
-the [Third Party Handoffs section](#third-party-handoffs) first.
-
-This certificate is provided by the gifter to the exporter, the exporter then
-creates a `desc:handoff-receive` which includes this `desc:handoff-give`
-certificate. The point of this certificate is to provide the exporter some
-crucial information that it needs to check the `desc:handoff-receive` message.
-
-How to check the validity of this certificate is covered in the
-[`desc:handoff-receive`](#desc-handoff-receive).
+This record is used in [Third Party Handoffs](#third-party-handoffs).
```

Comment:
There’s a lot of passive voice throughout this document as it stands, but case by case, it’s worth asking whether you can inject the actor into the sentence.
```suggestion
In [Third Party Handoffs](#third-party-handoffs), the Gifter creates a `desc:handoff-give` and sends it to the Receiver.
The Receiver uses the `desc:handoff-give` to redeem the gift from the Exporter.
```